SonarQube.Client/Api/V5_10/GetIssuesRequest.cs

```diff
@@ -88,9 +88,11 @@ private static IEnumerable<T> ReadFromProtobufStream<T>(Stream stream, MessagePa
         }
 
         private static SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
-            new SonarQubeIssue(FilePathNormalizer.NormalizeSonarQubePath(issue.Path), issue.Checksum, issue.Msg,
+            new SonarQubeIssue(issue.Key, FilePathNormalizer.NormalizeSonarQubePath(issue.Path), issue.Checksum, issue.Msg,
```

Comment:
The change in this line is causing the "low" code coverage (one uncovered line out of six). There aren't any tests for this legacy method, and I'm not intending to add any as part of this change.